cmd/controller/main.go

```diff
@@ -88,15 +66,14 @@ func main() {
 	}
 
 	go startPkgController(stopCh, cfg, logger, atomicLevel)
```

Comment:
can you inline the startPkgController function here? the split was only to keep them separate.

Reply:
Done